dist/s_copyright

```diff
@@ -55,10 +55,8 @@ fi
 
 trap 'rm -f $c1 $c2 $c3 $c4 $c5' 0 1 2 3 13 15
 
-year=`date +%Y`
```

Comment:
I think it might be better to set year="present" ... just in case we need the flexibility later.